ra/registration-authority.go

```diff
@@ -339,6 +386,16 @@ func (ra *RegistrationAuthorityImpl) NewCertificate(req core.CertificateRequest,
 		return emptyCert, err
 	}
 
+	// Check total certificate rate limiting
+	totalIssued, err := ra.getIssuanceCount()
+	if err != nil {
+		logEvent.Error = err.Error()
+		return emptyCert, err
+	}
+	if totalIssued >= ra.rlPolicies.TotalCertificates.Threshold {
+		return emptyCert, fmt.Errorf("Certificate issuance limit reached")
```

Comment:
We want this error to get bubbled up to the user and should be a 429 (Rate Limited). Please define a new error type in core/util.go and add the necessary plumbing in WFE's statusCodeFromError.